README.md

```diff
@@ -219,7 +221,7 @@ The `boltzmann.decorators.params` function takes an ajv schema and generates a d
 
 ### Tests
 
-Boltzmann uses tap for testing. (We're looking for a replacement.) It provides some convenience wrappers to make testing your route handlers easier. It provides [shot](https://github.com/hapijs/shot) as a way to inject requests into your service. If you're using postgres, it wraps each test in a transaction so you can exercise your db code without changing state in your underlying database. We feel that databases are not behavior we want to replicate in mocks. It's more useful to test use of the db directly.
+Boltzmann uses tap for testing. It provides some convenience wrappers to make testing your route handlers easier. It provides [shot](https://github.com/hapijs/shot) as a way to inject requests into your service. If you're using postgres, it wraps each test in a transaction so you can exercise your db code without changing state in your underlying database. We feel that databases are not behavior we want to replicate in mocks. It's more useful to test use of the db directly.
```

Comment:
Just noticed: we don't actually expose `shot` – we wrap it up as `assert.request`. Probably not necessary to change in this PR, since we'll probably want to write a guide on "how to test a boltzmann app".